web/src/components/tts/UsernameForm.js

```diff
@@ -1,13 +1,17 @@
-import React, {useState} from "react"
+import React, {useEffect, useState} from "react"
 
 const UsernameForm = ({channelsToListen, onUpdateTwitchUsername}) => {
-  const [localUsername, setLocalUsername] = useState(channelsToListen)
+  const [localUsername, setLocalUsername] = useState('')
```

Comment:
```suggestion
  const [localUsername, setLocalUsername] = useState(channelsToListen)
```

Дефолтный стейт из пропсов тоже можно было бы взять. А то что добавил `useEffect` это супер правильно